src/pages/vaults/VaultDepositWithdrawForm.tsx

```diff
@@ -61,8 +66,12 @@ type VaultDepositWithdrawFormProps = {
   onSuccess?: () => void;
 };
 
-// execute
-const ex = <T,>(fn: () => T) => fn();
+const INDEXER_LAG_ALLOWANCE = timeUnits.second * 4;
+const QUANTUM_COUNT = 1e6;
```

Comment:
can we move to `constants/numbers`

Reply:
lmao I went to add it and it was already there